COVERAGE.md

```
@@ -740,302 +295,160 @@
 
-## @tevm/bundler-cache
+## @tevm/esbuild
 
 ```bash
-pnpm --filter @tevm/bundler-cache run test:coverage
+pnpm --filter @tevm/esbuild run test:coverage
 ```
 
 % Coverage report from v8
 -------------------|---------|----------|---------|---------|-------------------
 File               | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)